plugin/trino-hive/src/main/java/io/trino/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -385,11 +386,12 @@ private ListenableFuture<Void> loadPartition(HivePartitionMetadata partition)
         InputFormat<?, ?> inputFormat = getInputFormat(configuration, schema, false);
         FileSystem fs = hdfsEnvironment.getFileSystem(hdfsContext, path);
         boolean s3SelectPushdownEnabled = shouldEnablePushdownForTable(session, table, path.toString(), partition.getPartition());
```

Comment:
I would recommend avoiding static import here to provide more context to the reader:

`shouldEnablePushdownForTable` -> `S3SelectPushdown.shouldEnable`
`shouldEnableSplits` -> `S3SelectPushdown.isSplittable`